software/py/bsg_manycore_link_gen.py

```diff
@@ -44,11 +44,12 @@ class bsg_manycore_link_gen:
     "/*********************************************************\n" \
     + " BSG Manycore Linker Script \n\n"
 
-  def __init__(self, default_data_loc, dram_size, imem_size, sp):
+  def __init__(self, default_data_loc, dram_size, imem_size, sp, move_rodata_to_dmem):
     self._default_data_loc = default_data_loc
     self._dram_size = dram_size
     self._imem_size = imem_size
     self._sp = sp
+    self._move_rodata_to_dmem = move_rodata_to_dmem
     self._opening_comment += \
```

Comment:
Maybe add a print comment for the new parameter?

Reply:
okay addressed!